### ISSUE TITLE
YAMLLoadWarning on import

### ISSUE THREAD

**Issue by soliverc** [jmcarpenter2/swifter#97]
My first two lines in a Jupyter notebook:
```python
import pandas as pd
import swifter
```

I get the error:
```python
C:\Users\seanc\Anaconda3\lib\site-packages\dask\config.py:168: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  data = yaml.load(f.read()) or {}
```
Something has been deprecated but I'm not sure what and I don't know how to fix it. Is this a big deal or will it cause problems?

**Comment by soliverc** [comment]
Also another warning when I tested it using apply:
```python
df['fixed_name'] = df['term_location'].swifter.apply(shop_name_fixer)

C:\Users\seanc\Anaconda3\lib\site-packages\tqdm\std.py:651: FutureWarning: The Panel class is removed from pandas. Accessing it from the top-level namespace will also be removed in the next version
  from pandas import Panel
```
It still works though, it's currently applying the function.

**Comment by jmcarpenter2** [comment]
Hi @soliverc,

Thanks for raising this issue. Those warnings appear to be coming from the underlying libraries `dask` and `tqdm`. Can you please let us know what version of each library you are using? Specifically, I am interested knowing what version you are using for the following four libraries:
* `pandas`
* `dask` 
* `tqdm` 
* `swifter`

Thanks,
Jason

**Comment by soliverc** [comment]
Pandas 0.25.0
Dask 1.1.4
tqdm 4.38.0
Swifter 0.297

**Comment by jmcarpenter2** [comment]
Hi @soliverc ,

Sorry for the delay! I did some digging to uncover where these warnings are coming from.

With regards to the first issue. This [issue](https://github.com/dask/distributed/issues/2564) noted the deprecated yaml load, and this [PR](https://github.com/dask/distributed/pull/2566) resolved it back in March, 2019. Please upgrade to dask>=1.26.1 to resolve the yaml warning.

With regards to the second issue, it appears that this was flagged in this [tqdm issue](https://github.com/tqdm/tqdm/issues/824) and resolved by this [PR](https://github.com/tqdm/tqdm/pull/834) back in October, 2019. However, that PR was merged on October 31, 2019, the day tqdm==4.37.0 was released. Since your version (4.38.0) is after that one, I am not sure why this warning is still appearing. Regardless, this is an issue upstream with tqdm and as such not one I can solve.

Thanks,
Jason
